test/parallel/test-module-binding.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 require('../common');
 const fixtures = require('../common/fixtures');
-const { internalModuleReadFile } = process.binding('fs');
+const { internalModuleReadJSON } = process.binding('fs');
 const { readFileSync } = require('fs');
 const { strictEqual } = require('assert');
 
-strictEqual(internalModuleReadFile('nosuchfile'), undefined);
-strictEqual(internalModuleReadFile(fixtures.path('empty.txt')), '');
-strictEqual(internalModuleReadFile(fixtures.path('empty-with-bom.txt')), '');
+strictEqual(internalModuleReadJSON('nosuchfile'), undefined);
+strictEqual(internalModuleReadJSON(fixtures.path('empty.txt')), '');
+strictEqual(internalModuleReadJSON(fixtures.path('empty-with-bom.txt')), '');
 {
```

Comment:
👆 These tests highlight the mixed signals being sent since they test `.txt` but the method is being shaped via #15767 to be `json` only.